AllReadyApp/Web-App/AllReady/Views/Campaign/Map.cshtml

```diff
@@ -22,7 +22,7 @@
     var modelCampaign = @Json.Serialize(Model);
 
     System.config({
```

Comment:
We seem to be duplicating this system configuration in a lot of files. Perhaps we could centralize it into one file. We could load it in the _Layout